src/app/components/dropdown/index.html

```diff
@@ -1,15 +1,176 @@
+            <div class="caption caption-success sky-text-success">Do use icon-only dropdowns when there is not space to include a label.</div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>  
+      <h3 class="sky-text-danger">Don't use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Don't use dropdowns as form inputs. Use HTML <sky-code>&lt;select&gt;</sky-code> fields instead.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/dropdown/usage-dont-input.png" />
+            <div class="caption caption-danger sky-text-danger">Don't use dropdowns as form inputs.</div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+    </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Anatomy">
+      <sky-docs-anatomy imagePath="~/assets/img/guidelines/dropdown/anatomy.png">
+        <sky-docs-anatomy-item anatomyNumber="1">Dropdown button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="2">Dropdown menu</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="3">Action</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Options">
+      <h3>Button types</h3>
+      <sky-demo-page-component-states>
+        <sky-demo-page-component-state stateName="Default" imagePath="~/assets/img/guidelines/dropdown/option-type-default.png"></sky-demo-page-component-state>
+        <sky-demo-page-component-state stateName="Context menu" imagePath="~/assets/img/guidelines/dropdown/option-type-context.png"></sky-demo-page-component-state>
+        <sky-demo-page-component-state stateName="Icon" imagePath="~/assets/img/guidelines/dropdown/option-type-icon.png"></sky-demo-page-component-state>
+      </sky-demo-page-component-states>
+        <h3>Button style</h3>
+        <div class="sky-text-block sky-margin-stacked-content-section">
+            <p>Dropdowns can use the default or primary style inherited from SKY UX buttons. See the <a routerLink="/design/guidelines/page-layouts" fragment="primary-action-guidelines">primary action guidelines</a> for details about when to use the primary style.</p>
+        </div>
+    </sky-demo-page-content>
+      
+    <sky-demo-page-content sectionHeading="Behavior and states">
+        <h3>Behavior</h3>
+        <div class="sky-text-block sky-margin-stacked-content-section">
+            <p>The behavior of the dropdown menu is defined by the <a routerLink="/components/popover">popover</a> component.</p>
+        </div>
+        <h3>States</h3>
+        <div class="sky-text-block sky-margin-stacked-content-section">
+            <p>Dropdowns inherit the styling for their different states (hover, disabled, etc.) from <a routerLink="/components/button">SKY UX buttons</a>.</p>
```

Comment:
It's minor, but I'd use a such as clause instead of parentheses. I'm going to make that change, but we can undo it if anyone objects.